index.html

```diff
@@ -1,299 +1,318 @@
-                    
-                </div>
-                <div class="letconnects">
-                <h4>LET’S CONNECT</h4>  
-                </div>
-                <div class="socialmediaclass">
-                    <img id="socialmedia" src="./img/socialmedia.png">
-                </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-menu">
+        <div class="sub-menu">
+          <h1>Samuel Kinyanjui</h1>
+          <ul>
+            <li id="down"><a href="#">Portfolio</a></li>
+            <li id="down"><a href="#">About</a></li>
+            <li id="down"><a href="#">Contact</a></li>
+          </ul>
+          <img src="./image/hamburguer1111.png" alt="humber" id="humb">
+        </div>
+      </nav>
+      <section class="main-bar">
+      <header>
+        <div class="sub-main">
+          <h1>I’m Samuel <br>
+            Glad to see you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or 
+            website Look through some of my work and experience! If you like what you 
+            see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="social">
+              <h4>LET’S CONNECT</h4>
+              <ul>
+                <li>
+                  <a href="#"
+                    ><img src="./image/twitter.png" alt="angellist"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./image/Linkedin icon.png" alt="Linkedin"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/medium.png" alt="github" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/github.png" alt="twitter" /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="./image/angellist.png" alt="medium" /></a>
+                </li>
+              </ul>
+            </div>
+        </div>
 
-            </section>
-        </section>
+      </header>
     </section>
+<!--cards-->
+    <section class="works">
+      <nav class="card" id="one">
+        <div id="img1"></div>
+        <div class="left-bloks" id="sub-left-block">
+          <div class="primary">
+            <h1>Tonic</h1>
+            <div class="frame">
+              <p id="font">CANOPY</p>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>Back End Dev</h6>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>2015</h6>
+            </div>
+          </div>
+          <div class="textsecond">
+          <p class="text">
```

Comment:
- [ ] You have been advise to make the font-size 16px, this rule has not been applied due to the cascading nature of css. Take a look at the picture below show that the rule is not applied. Please make sure that you apply that style last if you want it to take effect. 👍🏾 

![font](https://github.com/samuelkinyanjui/-finish_mobile_version_rev/assets/99656687/c8087b1b-ae61-4060-9ee3-aff49daf6d6a)